src/zcash/Zcash.h

```diff
@@ -28,4 +28,13 @@
 #define ZC_SAPLING_ENCCIPHERTEXT_SIZE (ZC_SAPLING_ENCPLAINTEXT_SIZE + NOTEENCRYPTION_AUTH_BYTES)
 #define ZC_SAPLING_OUTCIPHERTEXT_SIZE (ZC_SAPLING_OUTPLAINTEXT_SIZE + NOTEENCRYPTION_AUTH_BYTES)
 
+// ZIP 225 transaction sizes (V5 transaction format)
+#define ZC_ZIP225_BASE_TRANSACTION_SIZE 108
+#define ZC_ZIP225_BASE_ORCHARD_PROOF_SIZE 2720
+#define ZC_ZIP225_BASE_SIZE (ZC_ZIP225_BASE_TRANSACTION_SIZE + ZC_ZIP225_BASE_ORCHARD_PROOF_SIZE)
+// Variable size calculations are ultimately based on the number of nActionsOrchard sections
+#define ZC_ZIP225_VARIABLE_TRANSACTION_SIZE 884
+#define ZC_ZIP225_VARIABLE_ORCHARD_PROOF_SIZE 2272
+#define ZC_ZIP225_VARIABLE_SIZE (ZC_ZIP225_VARIABLE_TRANSACTION_SIZE + ZC_ZIP225_VARIABLE_ORCHARD_PROOF_SIZE)
```

Comment:
We can split up the constants to show how they are composed:
```suggestion
// - CompactSize is at least 1 byte
#define ZC_ZIP225_ORCHARD_NUM_ACTIONS_BASE_SIZE 1
#define ZC_ZIP225_ORCHARD_FLAGS_SIZE 1
#define ZC_ZIP225_ORCHARD_VALUE_BALANCE_SIZE 8
#define ZC_ZIP225_ORCHARD_ANCHOR_SIZE 32
// - CompactSize is at least 2 bytes because sizeProofsOrchard >= 253
#define ZC_ZIP225_ORCHARD_SIZE_PROOFS_BASE_SIZE 2
#define ZC_ZIP225_ORCHARD_PROOF_BASE_SIZE 2720
#define ZC_ZIP225_ORCHARD_BINDING_SIG_SIZE 64
#define ZC_ZIP225_ORCHARD_BASE_SIZE (ZC_ZIP225_ORCHARD_NUM_ACTIONS_SIZE + ZC_ZIP225_ORCHARD_FLAGS_SIZE + ZC_ZIP225_ORCHARD_VALUE_BALANCE_SIZE + ZC_ZIP225_ORCHARD_ANCHOR_SIZE + ZC_ZIP225_ORCHARD_SIZE_PROOFS_BASE_SIZE + ZC_ZIP225_ORCHARD_PROOF_BASE_SIZE + ZC_ZIP225_ORCHARD_BINDING_SIG_SIZE)
// Marginal transaction size per Orchard Action
#define ZC_ZIP225_ORCHARD_ACTION_SIZE 820
#define ZC_ZIP225_ORCHARD_SPEND_AUTH_SIG_SIZE 64
#define ZC_ZIP225_ORCHARD_PROOF_MARGINAL_SIZE 2272
#define ZC_ZIP225_ORCHARD_MARGINAL_SIZE (ZC_ZIP225_ORCHARD_ACTION_SIZE + ZC_ZIP225_ORCHARD_SPEND_AUTH_SIG_SIZE + ZC_ZIP225_ORCHARD_PROOF_MARGINAL_SIZE)
```

Reply:
Force-pushed to address review comments